### PR TITLE
Working `prefix_de`

### PR DESCRIPTION
**Pull request by maurolacy**
#432 follow-up. This separate PR implements / fixes `range_de` for a `prefix_de`.

### CODE REVIEW

**Comment by maurolacy** [comment]
Aand... fixed.

File: packages/storage-plus/src/de.rs [hunk 1]
```diff
@@ -0,0 +1,106 @@
+use crate::keys::IntKey;
```
Comment (ethanfrey) [inline packages/storage-plus/src/de.rs:1]: huh, I thought I saw this change in https://github.com/CosmWasm/cw-plus/tree/198-better-range-prefix already

this diff will be hard to review. maybe you can clean up the git history so the GitHub diff displayed is the diff I should review (maybe after merging the other PR #432)

Comment (maurolacy) [reply packages/storage-plus/src/de.rs:1]: Yes. GitHub got confused somehow with all the changes. If you do a local diff though, it looks OK.

**Comment by ethanfrey** [comment]
I see conflicting files. Seems it needs a rebase or merge or something.

**Comment by maurolacy** [comment]
Will fix / re-implement asap.

**Comment by ueco-jb** [comment]
> Will fix / re-implement asap.

Just remember you're on vacations

**Comment by maurolacy** [comment]
Merging this into #432's branch, for ease of rebasing.

**Comment by maurolacy** [comment]
> > Will fix / re-implement asap.
> 
> Just remember you're on vacations

Yeah, I know. I just can't help myself. Left this half-way through, and now can't avoid keeping an eye on it.

**Pull request merged**